tests/test_base.py

```diff
@@ -112,7 +113,7 @@ def test_itemgrader():
     with raises(ConfigError, match="Expected at least one answer in answers"):
         grader = StringGrader(answers=())
         grader(None, "hello")
-    with raises(ConfigError, match="Expected string for student_input, received <type 'int'>"):
+    with raises(ConfigError, match="Expected string for student_input, received {int}".format(int=int)):
```

Comment:
`type(int)`? Perhaps I'm getting these all confused, but this code seems weird to me...